test/commons/text/native-text-alternative.js

```diff
@@ -1,46 +1,47 @@
-describe('text.nativeTextAlternative', function() {
+describe('text.nativeTextAlternative', function () {
   var text = axe.commons.text;
+  var fixtureSetup = axe.testUtils.fixtureSetup;
   var queryFixture = axe.testUtils.queryFixture;
   var nativeTextAlternative = text.nativeTextAlternative;
 
-  it('runs accessible text methods specified for the native element', function() {
+  it('runs accessible text methods specified for the native element', function () {
     var vNode = queryFixture('<button id="target">foo</button>');
     assert.equal(nativeTextAlternative(vNode), 'foo');
   });
 
-  it('returns the accessible text of the first method that returns something', function() {
+  it('returns the accessible text of the first method that returns something', function () {
     var vNode = queryFixture(
       '<input id="target" type="image" alt="foo" value="bar" title="baz">'
     );
     assert.equal(nativeTextAlternative(vNode), 'foo');
   });
 
-  it('returns `` when no method matches', function() {
+  it('returns `` when no method matches', function () {
     var vNode = queryFixture('<div id="target">baz</div>');
     assert.equal(nativeTextAlternative(vNode), '');
   });
 
-  it('returns `` when no accessible text method returned something', function() {
-    queryFixture('<div class="baz">baz</div>');
+  it('returns `` when no accessible text method returned something', function () {
+    fixtureSetup('<div class="baz">baz</div>');
```

Comment:
Should replace this line and the querySelector for the class.

```js
    var div = queryFixture('<div class="baz" id="target">baz</div>');
```

Reply:
Updated in 9e4d1fdd41f947c111b9cf85783bc23f5ae2c5a9